ext.doc.mc2.template

```diff
@@ -8,67 +8,65 @@
     'version' 'all'
     'tags' [ 'extensions' ]
     'desc' 
-      <'
+<'
 This function does **marvelous things**. For example, EUCLIDIANDIVISION performs an euclidian division.
```

Comment:
Shouln't this be PleaseRenameFunction instead of EUCLIDIANDIVISION? Or maybe add a comment on the 'name' line stating that for the rest of this template the name 'EUCLIDIANDIVISION' will be used.

Reply:
done, with a few fixes in the other comment lines